bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=sriov-network-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.33.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-unknown
```

Comment:
Doesn't see right...

Reply:
your right please take a look on this one https://github.com/openshift/sriov-network-operator/pull/899

Reply:
I fix that one

@wizhaoredhat can you take a look please :)

Reply:
LGTM